Sources/update-strings/Commands/SortCommand.swift

```diff
@@ -8,6 +8,9 @@ struct SortCommand: ParsableCommand {
         commandName: "sort",
         abstract: "Sorts the contents of strings files in a given folder"
     )
+
+    @Option(name: .shortAndLong, help: "Option to enable validation mode. It throws an error if a file is not sorted.")
+    var validate = false
```

Comment:
I wonder if there's a different name for this. `--dry-run` is a term I've seen used for similar things. It'd also remove using `-v` as a flag which is usually for `verbose` or `version`.